rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -1087,6 +1076,15 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 
 		for (uint i = 0; i < items.size(); i++)
 		{
+			if (id >= 0)
```

Comment:
do this in the loop above, otherwise sorting is broken

Reply:
Fixed.